### DISCUSSION TITLE
Sonido surround?

### DISCUSSION THREAD

**Discussion by SuConde** [Paco8/plugin.video.movistarplus#46] [General]
Hola, 
Antes que nada gracias por el excelente trabajo!
Me preguntaba si se puede tener sonido digital (Dolby/DTS) 5.1 con el plugin. 
Mil gracias de nuevo! 
Juan Pablo

**Comment by Paco8** [comment]
Pues no sé, ¿hay sonido surround si reproduces desde la web de movistar?

**Comment by SuConde** [comment]
Desde el Chrome al menos no. Pero tampoco Netflix o Disney+ ofrecen surround desde el navegador, pero sí desde sus aplicaciones para Windows. Por eso preguntaba, ya que Kodi maneja muy cómodamente el surround

**Comment by Paco8** [comment]
¿La aplicación para android de movistar tiene sonido surround?

**Reply by SuConde** [reply]
Ni idea, estoy hablando siempre de Windows. Los programas/reproductores nativos tanto de Netflix como de Disney+ si transmiten en surround, pero accediendo desde el navegador web no. Movistar no tiene un reproductor para Windows, por eso me preguntaba si sí había manera de acceder al surround desde Windows usando Kodi u otro similar.
Nada, me enrollo mucho con el sonido siempre 🤣🤣
Igual mil gracias por el addon que funciona excelente! 😊😊

**Reply by Paco8** [reply]
Lo decía porque si la app de Movistar tiene surround en algún sistema entonces se podría investigar para ver si se le puede añadir al addon. Pero si no tiene entonces no hay nada que hacer.

**Comment by Paco8** [comment]
Parece que en el contenido 4K hay sonido DD+.

**Comment by SuConde** [comment]
Uff!! Si, ahi estoy chequeando, en 4k si esta, pero en muchas 1080 también. Aunque es muy random en cual sí y en cual no. (Dune no, Dune 2 si por ejemplo y están ambas en1080)
Pues nada, mil gracias por todo nuevamente! (y mas felicitaciones todavia!! :D que addon esta mas que perfecto!! :) )
